website/i18n/en.json

```diff
@@ -98,6 +98,9 @@
       "wdio-reportportal-service": {
         "title": "Report Portal Service"
       },
+      "wdio-rerun-service": {
+        "title": "Re-run Service Service"
```

Comment:
I guess you've not rebased with the latest master because it should be only one `Service` word here.

I can update en.json in my TypeScript PR